Gemfile

```diff
@@ -19,18 +19,18 @@ gem 'barnes' # heroku ruby-specific metrics
 gem 'bootstrap', '~> 4.5' # pin until major version is handled
 gem 'exception_notification'
 gem 'get_process_mem'
-gem 'gon', '~> 6.4.0'
+gem 'gon', '~> 6.4'
```

Comment:
I think this will match any minor version of 6.0, which may or may not be what you intended?

Reply:
It is! They should all be compatible, and most of the other constraints allow minor upgrades (the exceptions are rails and audited, where we have some custom patches / logic depending on internals, afaik, which we don't here)

Reply:
That makes sense! just wanted to make sure it was the intended behavior, that's tripped me up before.

Reply:
Makes sense! Thanks =)